### ISSUE TITLE
First children directories and files are not aligned.

### ISSUE THREAD

**Issue by cevhyruz** [nvim-tree/nvim-tree.lua#1440]
`first_child_dir1` and `first_child_file_1` should be aligned

![Screenshot from 2022-07-18 21-38-31](https://user-images.githubusercontent.com/20886353/179524082-5e1dc04c-d243-4661-912c-15d1c1671929.png)

Here is my setup:

```lua
nvim_tree.setup {
  actions = {
    open_file = {
      quit_on_open = true,
      window_picker = {
        enable = true
      }
    }
  },
  renderer = {
    indent_markers = {
      enable = true,
        icons = {
              edge = "│ ",
              item = "├─",
            corner = "╰─",
              none = " ",
          },
    },
    group_empty = false,
    add_trailing = false,
    root_folder_modifier = ":t",
    highlight_git = true,
    icons = {
      show = {
        git = true,
        folder = true,
        file = true,
        folder_arrow = true,
      },
      glyphs = {
        default = "",
        symlink = "",
        git = {
          unstaged = "",
          staged = "S",
          unmerged = "",
          renamed = "➜",
          deleted = "",
          untracked = "U",
          ignored = "◌",
        },
      },
      git_placement = "after"
    }
  },

  hijack_netrw = false,
  open_on_setup = false,
  ignore_ft_on_setup = {
    "startify",
    "dashboard",
    "alpha",
  },
  open_on_tab = false,
  hijack_cursor = false,
  update_cwd = false,
  diagnostics = {
    enable = true,
    icons = {
      hint = "",
      info = "",
      warning = "",
      error = "",
    },
  },
  update_focused_file = {
    ignore_list = {},
  },
  system_open = {
    cmd = nil,
    args = {},
  },
  filters = {
    dotfiles = false,
    custom = {},
  },
  git = {
    enable = true,
    ignore = false,
    timeout = 500,
  },
  view = {
    width = 35,
    height = 30,
    hide_root_folder = false,
    side = "left",
    preserve_window_proportions = true,
    mappings = {
      custom_only = false,
      list = {
        { key = { "l", "<CR>", "o" }, cb = tree_cb "edit" },
        { key = "h", cb = tree_cb "close_node" },
        { key = "v", cb = tree_cb "vsplit" },
        { key = '<Space>', action = 'edit' }
      },
    },
    number = false,
    relativenumber = false,
  },
  trash = {
    cmd = "trash",
    require_confirm = true,
  }
}

```

**Comment by cevhyruz** [comment]
It could be nice to support two-character indent markers though.

**Comment by cevhyruz** [comment]
actually It's still the same.

**Comment by kyazdani42** [comment]
i thought i fixed that already. I'll see for a patch in the next days

**Comment by alex-courtis** [comment]
Should be resolved via #1505

Please open an issue if further problems are experienced.
